web/packages/teleport/src/AppLauncher/AppLauncher.tsx

```diff
@@ -72,7 +72,11 @@ export function AppLauncher() {
       window.location.replace(`https://${fqdn}${port}${path}`);
     } catch (err) {
       let statusText = 'Something went wrong';
-      if (err instanceof Error) {
+
+      if (err instanceof TypeError) {
+        // `fetch` returns `TypeError` when there is a network error.
+        statusText = `Unable to access "${params.fqdn}" due to a network issue. Please ensure the domain is configured correctly, and your browser trusts the SSL certificate.`
```

Comment:
This is most commonly happens when your Teleport Proxy is using self-signed certificate correct? Can we just say so explicitly?
```suggestion
        statusText = `Unable to access "${params.fqdn}". This may happen if your Teleport Proxy is using untrusted or self-signed certificate. Please ensure Teleport Proxy service uses valid certificate or access the application domain directly (<give URL here>) and accept the certificate exception from your browser.`
```